sdk/bazel_tools/googleapis-status-proto.patch

```diff
@@ -6,7 +6,7 @@ index 9c7900ce7..a7a256a16 100644
  load("@rules_proto//proto:defs.bzl", "proto_library")
  load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
 
-+exports_files(["status.proto"])
++exports_files(["status.proto", "error_details.proto"])
```

Comment:
To read the information we want, we need to decode an `ErrorInfo` proto object, so this additional proto is needed